src/Estimators/OneBodyDensityMatrices.cpp

```diff
@@ -354,6 +361,82 @@ inline void OneBodyDensityMatrices::generateDensitySamples(bool save,
   rhocur_ = rho;
 }
 
+inline void OneBodyDensityMatrices::generateDensitySamplesWithSpin(bool save,
```

Comment:
generateDensitySamples has a unit test this can as well.

Reply:
fixed